aspnetcore/fundamentals/servers/aspnet-core-module.md

```diff
@@ -0,0 +1,85 @@
+ms.topic: article
+ms.assetid: 4661af33-34c5-4d71-93a0-8c7632f43580
+ms.prod: aspnet-core
+uid: fundamentals/servers/aspnet-core-module
+---
+# ASP.NET Core Module overview
+
+By [Tom Dykstra](http://github.com/tdykstra), [Rick Strahl](https://github.com/RickStrahl), and [Chris Ross](https://github.com/Tratcher) 
+
+ASP.NET Core Module (ANCM) is an IIS module that lets [Kestrel](kestrel.md) use IIS or IIS Express as a reverse proxy server. **ANCM works only with Kestrel; it isn't compatible with [WebListener](weblistener.md).** 
+
+Supported Windows versions:
+
+* Windows 7 and Windows Server 2008 R2 and later
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/fundamentals/servers/aspnet-core-module/sample)
+
+## What ASP.NET Core Module does
+
+ANCM is a native IIS module that hooks into the IIS pipeline very early in the request cycle and immediately redirects all traffic to the backend ASP.NET Core application. All requests - even those mapped to top level Handlers like ASPX - bypass the IIS pipeline and are forwarded to the ASP.NET Core application.
+
+Because ASP.NET Core applications run in a process separate from the IIS worker process, ANCM also does process management. ANCM starts the process for the ASP.NET Core application when the first request comes in and restarts it when it crashes. This is essentially the same behavior as classic ASP.NET applications that run in-process in IIS and are managed by WAS (Windows Activation Service).
+
+Here's a diagram that illustrates the relationship between IIS, ANCM, and ASP.NET Core applications.
+
+![ASP.NET Core Module](aspnet-core-module/_static/ancm.png)
+
+Requests come in from the Web and hit the kernel mode Http.Sys driver which routes into IIS on the primary port (80) or SSL port (443). The request is then forwarded to the ASP.NET Core application on the HTTP port configured for the application, which is not port 80/443. Kestrel picks up the request and pushes it into the ASP.NET Core middleware pipeline which then handles the request and passes it on as an `HttpContext` instance to application logic. The application's response is then passed back to IIS, which pushes it back out over the Internet to the HTTP client that initiated the request.
+
+ANCM has a few other functions as well:
+
+* Sets environment variables.
+* Logs `stdout` output to file storage.
+* Forwards Windows authentication tokens.
+
+## How to use ANCM in ASP.NET Core apps
+
+This section provides an overview of the process for setting up an IIS server and ASP.NET Core application. For detailed instructions, see [Publishing to IIS](../../publishing/iis.md).
+
+### Install ANCM
+
+The ASP.NET Core Module has to be installed in IIS on your servers and in IIS Express on your development machines. For servers, ANCM is included in the [ASP.NET Core Server Hosting Bundle](http://go.microsoft.com/fwlink/?LinkId=827547). For development machines, Visual Studio automatically installs ANCM.
```

Comment:
For development machines VS automatically installs ANCM for IIS Express and IIS (if currently installed).